lib/cadet/assessments/library.ex

```diff
@@ -1,22 +1,25 @@
 defmodule Cadet.Assessments.Library do
   @moduledoc """
-  The library entity represents a library to be used in a programming question.
+  The library entity represents a library to be used in a  question.
   """
   use Cadet, :model
 
+  alias Cadet.Assessments.Library.ExternalLibrary
+
   embedded_schema do
     field(:chapter, :integer, default: 1)
     field(:globals, {:array, :string}, default: [])
```

Comment:
Hmm in the frontend, the type of `globals` is array of "tuple" of string and any (`Array<[string, any]>`). Maybe we can represent it as a `:map`.

Reply:
ah yes i missed this